Lesson_28/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+   <head>
+      <meta charset="UTF-8">
+      <meta http-equiv="X-UA-Compatible" content="IE=edge">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <title>Products</title>
+      <link rel="stylesheet" href="./styles/style.css">
+      <script src="./main.js" type="module"></script>
+   </head>
+   <body>
+      <main>
+         <div class="container">
+            <button class="add-product-btn btn">Add new product</button>
+            <p class="success-message d-none">Success!</p>
+            <div class="product-list"></div>
+            <div class="modal-window d-none">
```

Comment:
For UX it's more efficient to use a fixed modal window

<img width="1403" alt="image" src="https://user-images.githubusercontent.com/24583373/230064250-b49571d4-d7a0-4f24-982c-6e44b0a9131d.png">